src/smokeTest/resources/application.properties

```diff
@@ -0,0 +1 @@
+test-url=${TEST_URL}
```

Comment:
are you sure this file is being used by test?
I had to add `@TestPropertySource("classpath:application.properties")` in send-letter smoke tests

Reply:
This is pretty much a copy-paste from the producer. I've already run a successful build on cnp with this code, so it has to be used.